### PR TITLE
Add, View and month indicator

### PR DESCRIPTION
**Pull request by blaskodaniel**

### CODE REVIEW

**Comment by blaskodaniel** [comment]
The month indicator has  removed from mini app.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# sn-react-typescript-boilerplate
+# sn-react-typescript-memoapp
```
Comment (zoltanbedi) [inline README.md]:
```suggestion
# sn-react-typescript-calendar
```

File: src/components/add-new-event.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react'
+import { createStyles, Fab, makeStyles, Theme } from '@material-ui/core'
+import AddIcon from '@material-ui/icons/Add'
+import { SharedContext } from '../context/shared-context'
+
+/**
+ * Style for component with Material UI
+ */
+const useStyles: any = makeStyles((theme: Theme) =>
```
Comment (zoltanbedi) [inline src/components/add-new-event.tsx]:
```suggestion
const useStyles = makeStyles((theme: Theme) =>
```

File: src/components/mainpanel.tsx [hunk 3]
```diff
@@ -110,6 +111,7 @@ const MainPanel: React.FunctionComponent = () => {
       })
 
       const groupedby = groupByDay(result.d.results, 'StartDate')
+      console.log(groupedby)
```
Comment (zoltanbedi) [inline src/components/mainpanel.tsx]:
```suggestion
```

File: src/components/new-dialog-body.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            try {
+              await repo.post({
+                contentType: CalendarEvent.name,
+                parentPath,
+                content,
+              })
+              sharedcontext.setRefreshcalendar(!sharedcontext.refreshcalendar)
+              handleClose()
+            } catch (error) {
+              console.log('Error')
```
Comment (zoltanbedi) [inline src/components/new-dialog-body.tsx]:
```suggestion
              console.log(error.message)
```

**Review by zoltanbedi — CHANGES_REQUESTED** [review]
Minor things I found. Other than that it looks good 😎

**Comment by blaskodaniel** [comment]
Thanks

**Pull request merged**